src/main/kotlin/com/fatimacarneiro/estudokotlin/kafka/PhoneNumberProducer.kt

```diff
@@ -0,0 +1,43 @@
+package com.fatimacarneiro.estudokotlin.kafka
+
+import com.fatimacarneiro.estudokotlin.phonenumber.PhoneNumber
+import org.apache.kafka.clients.producer.KafkaProducer
+import org.apache.kafka.clients.producer.ProducerConfig
+import org.apache.kafka.clients.producer.ProducerRecord
+import org.apache.kafka.common.serialization.StringSerializer
+import org.springframework.stereotype.Repository
+import java.util.*
+
+import com.google.gson.Gson
+
+@Repository
+class PhoneNumberProducer {
+
+    fun producer(phoneNumber: PhoneNumber): KafkaProducer<String, String?> {
+        val producer = KafkaProducer<String, String?>(properties())
+        producer.send(record(phoneNumber))
```

Comment:
this works, but perhaps you can do somethings to improve:
- Use kafkaTemplate instead KafkaProducer. Changing this, you don't need to create the Record object.
- You can add serialize directly on properties. Using this, you don't need to convert objects to string manually. 
- You can separate this class in a configuration class and create kafkaTemplate as a bean
- Server and topic name is better to put on application.yml. 

You can see this way on my kafkaSpring project: https://github.com/iundarigun/kafka-series/tree/master/spring-kafka